Dockerfile

```diff
@@ -1,28 +1,25 @@
-FROM rocker/binder:4.3.2
-
+FROM rocker/binder:latest@sha256:9c1bb3dc842755c4ac57b6e5ab78dd353c0f4790bdcd7d1f780b6dff38435d9a
 USER root
 
 COPY requirements.txt /tmp/requirements.txt
 RUN pip install --no-cache -r /tmp/requirements.txt
 
-# Install packages needed for quarto knitting to PDF
-RUN tlmgr install \
-    koma-script \
-    mdwtools \
-    tikzfill \
-    bookmark
+RUN apt-get update && apt-get -y install \
```

Comment:
I would suggest we stick with `tlmgr`, as that's what R uses to install tex packages needed at runtime. This used to come from apt, and that caused so many issues.